docs/configuration/index.md

```diff
@@ -955,6 +956,7 @@ Issuing a GET request at the same URL will return the spec that is currently in
 |`replicationThrottleLimit`|The maximum number of segment replicas that can be assigned to a historical tier in a single Coordinator run. This property prevents historicals from becoming overwhelmed when loading extra replicas of segments that are already available in the cluster.|500|
 |`balancerComputeThreads`|Thread pool size for computing moving cost of segments during segment balancing. Consider increasing this if you have a lot of segments and moving segments begins to stall.|1|
 |`killDataSourceWhitelist`|List of specific data sources for which kill tasks are sent if property `druid.coordinator.kill.on` is true. This can be a list of comma-separated data source names or a JSON array.|none|
+|`killTaskSlotRatio`| Ratio of total available task slots, including autoscaling if applicable that will be allowed for kill tasks. This limit only applies for kill tasks that are spawned automatically by the coordinator's auto kill duty, which is enabled when `druid.coordinator.kill.on` is true.| none - no limit                |
```

Comment:
Instead of dynamic configuration, should we have a dedicated auto-kill config similar to [auto-compaction](https://druid.apache.org/docs/latest/operations/api-reference.html#automatic-compaction-configuration)? I think that will ease [config management](https://druid.apache.org/docs/latest/operations/api-reference.html#automatic-compaction-status) a bit. We can then move all these kill* properties to the dedicated kill config. What do y'all think?

Reply:
I considered this as well 😆 . I thought since other kill properties are already here at this level to expand and add more. I frankly didnt want to have to deal with the resolution of properties from multiple possible configs, deprecated and new, so decided against it. If you feel strongly let me know. And lets see what others thing as well.